### ISSUE TITLE
Gui theme: text color dissociation between object and message

### ISSUE THREAD

**Issue by 60-hz** [pd-l2ork/pd-l2ork#10]
I would like to make a theme in order to reduce the classic message / object color confusion with students.

But in .css files, It is currently not possible to choose a different color between text inside object boxes and text inside message box, which don't allow that kind of theme using white text for boxes and black text for messages and comment:

![Capture_d_écran_2020-10-05_à_12 20 42](https://user-images.githubusercontent.com/5449719/204806074-4531e912-def0-4749-9434-74427bd0fc22.png)

**Comment by 60-hz** [comment]
The mix blend mode is a nice idea, and it prevent user to choose fancy unreadable colors matches.

Great if the theme could be added in pd-l2ork. During pd Workshops, I use another signal colors with pd-next, which allows also a default color for graphic array:
<img width="662" alt="Capture d’écran 2022-11-30 à 16 27 14" src="https://user-images.githubusercontent.com/5449719/204841371-8b731fd8-b79c-4014-b4ec-a1681a3d312f.png">

**Comment by pd-l2ork** [comment]
Also, do you have a css file you may have edited to match as close to this as possible?

**Comment by pd-l2ork** [comment]
Nice. Please reuse orange that we already have in the default.css file. We can also investigate making signal cords more Max-like with orange/black (or perhaps two hues of orange?) stripes, which is just another tiny CSS modification. See: https://css-tricks.com/stripes-css/

**Comment by 60-hz** [comment]
Since orange is less visible  on top of grey background, I put red for signal. Also orange is already used in selections and temporary cords mouse drawing... still can be perfected but that's a start.
Also the grid color of edit mode is making things difficult to read in many themes, I don"t see the use of drawing it actually since the snapping is perfect to organise boxes in patch and make the user "feel the grid" rather than actually showing it.
But the interesting use of the grid is to maker the user aware that the patch is in edit mode, so then it could be drawn lighter or, better, with bigger cases?

Here is a screenshot with "simulated" white text on grey objects.

<img width="1004" alt="Capture d’écran 2022-11-30 à 19 30 05" src="https://user-images.githubusercontent.com/5449719/204880575-3ae813f6-463e-4714-9eba-f7059ba6f27b.png">

**Comment by pd-l2ork** [comment]
I like it a lot. How does the orange selection color coexist with red and other colors? Is it visible and discernible enough? Can you post a screenshot of that where you select some or all objects on these patches?

I also see that in the documentation patch the inlet number looks like it is blending too much with the background, so we would have to go through all the documentation and edit the color. Perhaps we could create a script for this purpose?

**Comment by pd-l2ork** [comment]
Also, what if we used cyan for signal cords like those arrows on K12 objects?

**Comment by 60-hz** [comment]
Orange selection looks like that...

<img width="219" alt="Capture d’écran 2022-12-01 à 11 00 26" src="https://user-images.githubusercontent.com/5449719/205023952-c9011934-4333-42f6-ae4b-1539487d0f9b.png">

Maybe I can try cyan for selection?
The biggest confusing stuff is also grid density.

**Comment by 60-hz** [comment]
>I also see that in the documentation patch the inlet number looks like it is blending too much with the background, so we would have to go through all the documentation and edit the color. Perhaps we could create a script for this purpose?

Oh yes, I didn't notice this... I tried to turn the background ligther but then messages/atom boxes are not visible :/

**Comment by pd-l2ork** [comment]
So, I just uploaded a new version of pd-l2ork that includes a new 60-hz theme :-). I did some editing while trying to stay as close to the original. E.g. comments when selected are hard to read. Red cords are not as good of an idea (IMO) because red within the context of pd-l2ork typically denotes something broken. Lastly, added custom color options for default arrays, background (small and larger tile lines), and customizable object text color. See 60-hz.css for more info. Lemme know if it all works ok.
